css/fieldmanager.css

```diff
@@ -30,6 +47,44 @@
 	margin: 0 0 0 5px;
 }
 
+.fm-label label {
+	font-size: 13px;
+	font-weight: 700;
+	line-height: 1.4;
+	display: inline-block;
+	margin: 0 0 4px;
+}
+
+.fmjs-removable-label label {
+	line-height: 1.7em;
+}
+
+/* Each FM Field */
```

Comment:
Consider using a more prominent section divider comment if this refers to a large chunk of what follows. Perhaps [Idiomatic CSS comment style](https://github.com/necolas/idiomatic-css#comments) would be useful? Example from the linked page:

```css
/* ==========================================================================
   Section comment block
   ========================================================================== */

/* Sub-section comment block
   ========================================================================== */

/**
 * Short description using Doxygen-style comment format
 *
 * The first sentence of the long description starts here and continues on this
 * line for a while finally concluding here at the end of this paragraph.
 *
 * The long description is ideal for more detailed explanations and
 * documentation. It can include example HTML, URLs, or any other information
 * that is deemed necessary or useful.
 *
 * @tag This is a tag named 'tag'
 *
 * TODO: This is a todo statement that describes an atomic task to be completed
 *   at a later date. It wraps after 80 characters and following lines are
 *   indented by 2 spaces.
 */

/* Basic comment */
```

Reply:
I agree although the basic comments here are just following the pattern in the rest of the stylesheets.